src/ophys_etl/transforms/data_loaders.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+import numpy as np
+from collections import namedtuple
+
+motion_border = namedtuple('motion_border', 'left right up down')
+
+
+def get_max_correction_values(motion_corr_df: pd.DataFrame,
+                              max_shift: float = 30.0) -> motion_border:
+    """
+    Gets the max correction values in the cardinal directions from
+    a motion correction dataframe created during Motion Correction workflow
+    step.
+    Parameters
+    ----------
+    motion_corr_df: pd.DataFrame
+        The dataframe that contains the motion correction value calculated
+        during motion correction workflow step of ophys segmentation pipeline.
+    max_shift: float
```

Comment:
Is this lifted from the SDK?

Reply:
Mostly, I made some slight changes.

Reply:
The logic is all the same however

Reply:
Can you point me to the SDK functions? thanks!

Reply:
`allensdk\internal\brain_observatory\roi_filter_utils.py::calculate_max_border` these values are then assigned to the ROIs in the LIMS strategy as the max correction values after this module has run but before uploading to LIMS

Reply:
There's no default value for max_shift in that function. How did you decide on 30?

Reply:
The 30 shift is defined as global variable in the run_roi_filter module that LIMs strategy calls. That is where I got the 30 pixel default from. You can see this defined at `allensdk\internal\pipeline_modules\run_roi_filter.py` as MAX_SHIFT